course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -531,26 +560,218 @@
+
+            $('#'+divID).html($compile(angular.element(cal_button))($scope));
+        };
+
+        // Converts the mm/dd/yyyy date to a yyyy-MM-ddT00:00:01:00Z format
+        $scope.formatConcludeDate = function(date) {
+            var concludeDate = null;
+            if (date) {
+                concludeDate = $filter('date')(new Date(date), 'yyyy-MM-dd', 'Z')+'T00:01:00Z';
+            }
+            return concludeDate;
+        };
+
+        // Checks if the given date is prior to today's date.
+        $scope.isSelectedDateInPast = function(selectedDate) {
+            // Since the input field is a string representation of a date,
+            // we need to convert today's date to the same format as a string to make the comparison.
+            var today = new Date();
+            var todayString = (today.getMonth() + 1) + '/' + today.getDate() + '/' +  today.getFullYear();
+            return Date.parse(selectedDate)-Date.parse(todayString)<0;
+        };
+
+        // Perform the PATCH with the given user ID, role_id, conclude_date
+        $scope.updateConcludeDate = function(userID, roleID, concludeDate) {
+            var patchData = {
+                'user_id': userID,
+                'role_id': roleID,
+                'conclude_date': $scope.formatConcludeDate(concludeDate)
+            };
+            var url = djangoUrl.reverse('icommons_rest_api_proxy',
+                                        ['api/course/v2/course_instances/'
+                                         + $scope.courseInstanceId + '/people/' + userID + '/']);
+
+            $http.patch(url, patchData)
+                .success(function finalizeCourseDetailsPatch() {
+                    // If update is successful, then touch the course instance so the last_updated field is updated
+                    // so that changes will be picked up in feed
+                    $scope.updateCourseInstanceLastUpdated();
+                    // Reset the display to show conclude date and calendar button
+                    $scope.createCalButton(patchData['user_id'], concludeDate);
+                    $scope.addPopOverToCell(userID, 'success', 'The enrollment details have been updated.');
+                })
+                .error(function(data, status, headers, config, statusText) {
+                    $scope.handleAjaxError(data, status, headers, config, statusText);
+                    $scope.addPopOverToCell(userID, 'failure', 'An error occurred during the update.');
+                })
+        };
+
+        // Updates the current course instances 'last_updated' field to be the current date and time.
+        $scope.updateCourseInstanceLastUpdated = function() {
+            var url = djangoUrl.reverse('icommons_rest_api_proxy',
+                                        ['api/course/v2/course_instances/'
+                                         + $scope.courseInstanceId + '/']);
+            $http.patch(url);
+        };
```

Comment:
This line had me verify if the update_date on course_enrollee/staff/student gets updated when the conclude_date is modified by the  rest api call on line 748, and it doesn't seem to be. .  I made a comment on the other PR that is closed, just cross referencing it here
https://github.com/Harvard-University-iCommons/icommons_rest_api/pull/114/files#diff-2de6c0d199fd14c7c2ed343274ad367fR1321

Reply:
This call will only update the last_updated field on a course instance. 
I will add a trigger for update on the enrollee/staff/student so the update_date field gets updated.
I am going to redeploy my branch to dev as the commit containing this call was added after my last deploy.